### DISCUSSION TITLE
Rename the window titles for "Live Timing" and "Race Control Messages" from "MultiViewer for F1" to representative versions

### DISCUSSION THREAD

**Discussion by Psybertr0n** [f1multiviewer/issue-tracker#93] [Feature requests] [answered]
At present the windows for "Live Timing" and "Race Control Messages" (after popping it out of live timing) are both named "MultiViewer for F1". Can the window titles for these be renamed to what they specifically represent? This would make it easier to spot them in the Windows task bar. It would also open up these windows to be detected via our own scripts and Elgato Stream Deck plugins. ;)

**Comment by f1multiviewer** [comment, marked as answer]
Will be done in the next release!
